File: pku/2864.c
#include <stdio.h>
int main(){int i,j,n,d,v,a[101];for(;scanf("%d%d",&n,&d),n;printf("%s\n",n-i?"yes":"no")){for(i=0;i<101;i++)a[i]=0;for(i=0;i<d;i++)for(j=0;j<n;j++)scanf("%d",&v),a[j]+=v;for(i=0;i<n;i++)if(a[i]==d)break;}return 0;}

File: pku/2864.java
import java.util.*;
import static java.lang.Math.*;

public class Main{
    public static void main(String[] args){
        new Main().run();
    }
    void run(){
        Scanner sc = new Scanner(System.in);
        for(;;){
            int n = sc.nextInt();
            int d = sc.nextInt();
            if(n+d == 0)break;
            int[] a = new int[n];
            for(int i = 0; i < d; i++)
                for(int j = 0; j < n; j++){
                    int v = sc.nextInt();
                    if(v!=0)a[j]++;
                }
            boolean f = false;
            for(int i = 0; i < n; i++)
                if(a[i] == d){
                    f = true;
                    break;
                }
            System.out.println(f ? "yes" : "no");
        }
    }
}
